wp-content/themes/core/blocks/tribe/terms/style.pcss

```diff
@@ -0,0 +1,82 @@
+/**
+ * The following styles get applied both on the front of your site
+ * and in the editor.
+ *
+ * Replace them with your own styles or remove the file completely.
+ */
+
+/* -----------------------------------------------------------------------
+ * Terms Block
+ * ----------------------------------------------------------------------- */
+
+.wp-block-tribe-terms {
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+
+	&.aligncenter {
+		justify-content: center;
+	}
+
+	&.alignright {
+		justify-content: flex-end;
+	}
+}
```

Comment:
Are there existing classes/styles we could use for this?

Reply:
Not that I know of. `aligncenter` and `alignright` come from the the alignment controls on the block and aren't "custom" to this block.